### ISSUE TITLE
[revolutiontt]  (Config)

### ISSUE THREAD

**Issue by ERR1R** [Jackett/Jackett#5695]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.

If you are experiencing an issue with a tracker, then:
1. Use your browser to check you can access the site directly, and if a login is required, check you can login and that you do not have any outstanding account issues.
2. If you haven't already, try upgrading to the latest version of Jackett, your issue may have already been resolved.
3. If it is still not working for you, then a **full enhanced log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

(At the bottom of your Jackett Dashboard page) **Jackett version**:  
(if you installed using Jackett.Binaries.Mono.tar.gz) **Mono version**:

-------------------------------
When editing credentials or adding the tracker i get the error.. Also when i searched for releases when it was added before i got 0 results.. So i thought i am going to update jackett and add the rev tracker again.. But i can't..

Jackett Version 0.11.502.0

Mono JIT compiler version 5.18.0.240

log

Jackett.Common.ExceptionWithConfigData
  at Jackett.Common.Indexers.RevolutionTT+<>c__DisplayClass16_0.<ApplyConfiguration>b__0 () [0x00034] in <13fe90e9c320497a97d69ba91cf6f629>:0 
  at Jackett.Common.Indexers.BaseIndexer.ConfigureIfOK (System.String cookies, System.Boolean isLoggedin, System.Func`1[TResult] onError) [0x00034] in <13fe90e9c320497a97d69ba91cf6f629>:0 
  at Jackett.Common.Indexers.RevolutionTT.ApplyConfiguration (Newtonsoft.Json.Linq.JToken configJson) [0x00231] in <13fe90e9c320497a97d69ba91cf6f629>:0 
  at Jackett.Server.Controllers.IndexerApiController.UpdateConfig (Jackett.Common.Models.DTO.ConfigItem[] config) [0x0012d] in <fadc3f4679f8435a82eee997394576d3>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor+TaskOfIActionResultExecutor.Execute (Microsoft.AspNetCore.Mvc.Infrastructure.IActionResultTypeMapper mapper, Microsoft.Extensions.Internal.ObjectMethodExecutor executor, System.Object controller, System.Object[] arguments) [0x00071] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync () [0x00131] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync () [0x0009e] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow (Microsoft.AspNetCore.Mvc.Filters.ActionExecutedContext context) [0x0001b] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next (Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+State& next, Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted) [0x00382] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync () [0x0002f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter () [0x0009f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow (Microsoft.AspNetCore.Mvc.Filters.ResourceExecutedContext context) [0x0001b] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next (Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+State& next, Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted) [0x00840] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync () [0x0002f] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync () [0x0012e] in <b4ef600f4a594fe2865a8f97f915fb9d>:0 
  at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext httpContext) [0x001cb] in <6092a16d93814eba828b517a2b132f80>:0 
  at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext context) [0x00384] in <427697fe42b7459ba5302fb76d339d3b>:0 
  at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke (Microsoft.AspNetCore.Http.HttpContext context) [0x0043e] in <f352e566abf6421e87eafbcf57a0b237>:0 
  at Jackett.Server.Middleware.CustomExceptionHandler.Invoke (Microsoft.AspNetCore.Http.HttpContext httpContext) [0x0008a] in <fadc3f4679f8435a82eee997394576d3>:0

**Comment by garfield69** [comment]
my suggestion is to try switching to the .net core version of jackett and ditch the mono dependent  version.
if you are using the jackett service, then don't forget to shutdown the service and delete the jackett mono startup script.

pick whichever jackett flavour fits your platform:
https://github.com/Jackett/Jackett#install-on-linux-amdx64
https://github.com/Jackett/Jackett#install-on-linux-armv7-or-above
https://github.com/Jackett/Jackett#installation-on-macos

**Comment by ERR1R** [comment]
Thanks.. i use synology. I used the beta but the version in the repo is not the standalone version.

I downloaded the the linux amd 64 and run jackett. It works but i cant add revo as tracker.. 

07-20 22:44:32 Error Jackett.Common.ExceptionWithConfigData
   at Jackett.Common.Indexers.RevolutionTT.<>c__DisplayClass16_0.<ApplyConfigura                                        tion>b__0() in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Revol                                        utionTT.cs:line 203
   at Jackett.Common.Indexers.BaseIndexer.ConfigureIfOK(String cookies, Boolean                                         isLoggedin, Func`1 onError) in /home/appveyor/projects/jackett/src/Jackett.Commo                                        n/Indexers/BaseIndexer.cs:line 267
   at Jackett.Common.Indexers.RevolutionTT.ApplyConfiguration(JToken configJson)                                         in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/RevolutionTT.cs:                                        line 198
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[]                                         config) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/Indexe                                        rApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResult                                        Executor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor,                                         Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMeth                                        odAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextAction                                        FilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionEx                                        ecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next                                        , Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilte                                        rAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter                                        ()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecuted                                        Context context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope&                                         scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsyn                                        c()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpConte                                        xt)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpCo                                        ntext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext c                                        ontext)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpCo                                        ntext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomEx                                        ceptionHandler.cs:line 29 Jackett.Common.ExceptionWithConfigData
   at Jackett.Common.Indexers.RevolutionTT.<>c__DisplayClass16_0.<ApplyConfigura                                        tion>b__0() in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Revol                                        utionTT.cs:line 203
   at Jackett.Common.Indexers.BaseIndexer.ConfigureIfOK(String cookies, Boolean                                         isLoggedin, Func`1 onError) in /home/appveyor/projects/jackett/src/Jackett.Commo                                        n/Indexers/BaseIndexer.cs:line 267
   at Jackett.Common.Indexers.RevolutionTT.ApplyConfiguration(JToken configJson)                                         in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/RevolutionTT.cs:                                        line 198
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[]                                         config) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/Indexe                                        rApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResult                                        Executor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor,                                         Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMeth                                        odAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextAction                                        FilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionEx                                        ecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next                                        , Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilte                                        rAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter                                        ()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecuted                                        Context context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope&                                         scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsyn                                        c()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpConte                                        xt)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpCo                                        ntext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext c                                        ontext)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpCo                                        ntext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomEx                                        ceptionHandler.cs:line 29
